pkg/controller/controller.go

```diff
@@ -624,6 +669,25 @@ func (p *csiProvisioner) Delete(volume *v1.PersistentVolume) error {
 	return err
 }
 
+func (p *csiProvisioner) SupportsBlock() bool {
+	ctx, cancel := context.WithTimeout(context.Background(), p.timeout)
+	defer cancel()
+
+	client := csi.NewControllerClient(p.grpcClient)
+	req := csi.ValidateVolumeCapabilitiesRequest{
+		VolumeCapabilities: []*csi.VolumeCapability{
+			volumeCapabilityMountReadWriteOnce,
```

Comment:
we don't seem to use this function anywhere. also I think it should be volumeCapabilityBlockReadWriteOnce

Reply:
@wongma7 

>  also I think it should be volumeCapabilityBlockReadWriteOnce

Thank you for pointing it out. It's a typo. I fixed it and modified unit code to detect this error.

> we don't seem to use this function anywhere. 

```SupportsBlock``` is called from [library](https://github.com/kubernetes-incubator/external-storage/blame/master/lib/controller/controller.go#L1319), as we added in the [commit](https://github.com/kubernetes-incubator/external-storage/pull/830), a bit long ago.